db/migrate/20170517233803_add_figure_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddFigureToPieces < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pieces, :figure, :string
```

Comment:
Ah, i see. I think you can use figure if you prefer that over type :) you just have to let Ruby know. I read that on one of the articles Jacob sent along. 

I was thinking if we would used the number, it would only be to distinguish initial position-- and the x_position/y_position can be used for that? For move logic and capture logic, I don't think it would be advantageous to use a number to decipher which move the piece can /cannot make. What is your vision on the use?